staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go

```diff
@@ -68,8 +69,9 @@ type CertificateOptions struct {
 	csrNames    []string
 	outputStyle string
 
-	clientSet clientset.Interface
-	builder   *resource.Builder
+	certificatesV1Client      v1.CertificatesV1Interface
+	certificatesV1Beta1Client v1beta1.CertificatesV1beta1Interface
```

Comment:
I double checked and it looks like certificates/v1beta1 is deprecated since 1.19, given our current supported skew it's about time we drop support for v1beta1. Add a new commit dropping that beta support and only then do your refactor. It'll be less code :smile:

Reply:
See https://github.com/kubernetes/api/blob/v0.25.0-beta.0/certificates/v1beta1/types.go#L30